src/CR.WorkingDayService/IWorkingDaySource.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="IWorkingDaySource.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.WorkingDayService
+{
+    using System;
+
+    /// <summary>
+    /// A common interface for implementing a Source of information on days' working/non-working status.
+    /// </summary>
+    public interface IWorkingDaySource
```

Comment:
Is there a benefit to having this as an interface over a base class? With the base class we could have the implementation of `IsNonWorkingDay` implemented directly rather than as an extension method.